exp_legacy/module/modules/gui/bonus.lua

```diff
@@ -113,58 +121,16 @@ local bonus_gui_control_pts_a = Gui.element("bonus_gui_control_pts_a")
         width = config.gui_display_width["half"],
     }
 
-local bonus_gui_control_pts_a_count = Gui.element("bonus_gui_control_pts_a_count")
-    :draw{
-        type = "label",
-        name = Gui.property_from_name,
-        caption = config.pts.base,
-        style = "heading_2_label",
-    }:style{
-        width = config.gui_display_width["half"],
-    }
-
---- Control label for the bonus points needed
--- @element bonus_gui_control_pts_n
-local bonus_gui_control_pts_n = Gui.element("bonus_gui_control_pts_n")
+local bonus_gui_control_pts_count = Gui.element("bonus_gui_control_pts_count")
     :draw{
-        type = "label",
+        type = "progressbar",
         name = Gui.property_from_name,
-        caption = { "bonus.control-pts-n" },
-        style = "heading_2_label",
-    }:style{
-        width = config.gui_display_width["half"],
-    }
-
-local bonus_gui_control_pts_n_count = Gui.element("bonus_gui_control_pts_n_count")
-    :draw{
-        type = "label",
-        name = Gui.property_from_name,
-        caption = "0",
-        style = "heading_2_label",
-    }:style{
-        width = config.gui_display_width["half"],
-    }
-
---- Control label for the bonus points remaining
--- @element bonus_gui_control_pts_r
-local bonus_gui_control_pts_r = Gui.element("bonus_gui_control_pts_r")
-    :draw{
-        type = "label",
-        name = Gui.property_from_name,
-        caption = { "bonus.control-pts-r" },
-        style = "heading_2_label",
-    }:style{
-        width = config.gui_display_width["half"],
-    }
-
-local bonus_gui_control_pts_r_count = Gui.element("bonus_gui_control_pts_r_count")
-    :draw{
-        type = "label",
-        name = Gui.property_from_name,
-        caption = "0",
-        style = "heading_2_label",
+        caption = "0 / 0",
+        value = 0,
+        style = "electric_satisfaction_statistics_progressbar",
     }:style{
         width = config.gui_display_width["half"],
+        font = "heading-2",
     }
```

Comment:
Need to add `color` here so that the bar is not green when the limit is exceeded. 
```suggestion
    }:style{
        width = config.gui_display_width["half"],
        font = "heading-2",
        color = { 1, 0, 0 },
    }
```